### ISSUE TITLE
Add "Preview" button to Platform pages in CMS

### ISSUE THREAD

**Issue by haleyjohnso** [vigetlabs/npm#337]
## Overview

- **URL:** https://www.nationalpublicmedia.com/platforms/audio/
- **Size:** 6
- **Level of urgency:** [Low]
- **Potential deadline:** [6/15]
- **Harvest task:** National Public Media [NPM] > [01] Redesign & Support

**Page or Element Name**
Wordpress editing interface of Platform pages

**Example URL**
https://www.nationalpublicmedia.com/platforms/audio/
	
## Details

**Summary:**
Add a "preview changes" button to the platform pages so that we can preview page edits before publishing. 

**Current vs. Desired state:**
Currently, these specific pages only have the option to "Update" (button at bottom of editing interface). We would like to be able to "preview" changes before publishing.

**Comment by ten1seven** [comment]
This work is staged up on staging as "phase 1".

New platform pages can be created here:
https://npm.staging.vigetx.com/wp/wp-admin/edit.php?post_type=platforms

Example:

New page:
https://npm.staging.vigetx.com/platforms-new/audio/

Old page:
https://npm.staging.vigetx.com/platforms/audio/

- **Phase 1** is to deploy this change up to production and recreate all the platform pages (only the top-level ones represented in the main nav are necessary). They will live at `/platforms-new`.
- **Phase 2** is to deploy an update that will turn off the ability for the platform taxonomy to have pages, remove the custom fields, and change the root slug on the new post type from `/platforms-new` to `/platforms`. This will all happen simultaneously with the deploy so there are no intermediary steps.

These steps have to happen on production (but can be tested on staging, of course) so that the new post types can be created on production.

**Comment by haleyjohnso** [comment]
I was able to QA this by creating this page https://npm.staging.vigetx.com/platforms-new/video/ and looked at it on a variety of screens and browsers. Does this mean we need to do phase 2 on staging as well?

**Comment by ten1seven** [comment]
Yep, we can do that to test phase 2. Keep in mind that because of the way Pantheon deploys work, we can't do phase 1&2 on test because anything on test goes to production. So we'll have to do phase 1 on _both_ before we can do phase 2.

**Comment by ten1seven** [comment]
@haleyjohnso phase 2 has been deployed to staging. You can now see the Video page you created now lives at `/platforms` https://npm.staging.vigetx.com/platforms/video/

**Comment by haleyjohnso** [comment]
@ten1seven I QA-ed those pages, but noticed that the links in the main navigation that should go to these pages now 404. Do I also need to update the links in the menu?

**Comment by ten1seven** [comment]
Correct, one of the steps is to switch over the menu links.

**Comment by haleyjohnso** [comment]
@ten1seven - Hoping to get this and the remaining tickets in the batch deployed next week. 

NPM mentioned that they'd be willing to do content entry. Here are the instructions I'm planning to send to them to explain the process--I bolded an assumption. Are my instructions accruate?:

1. First, we'll deploy a change that will allow you to create the new platform page types in the admin. 
2. Then, your team can recreate all the platform pages (only the top-level ones represented in the main nav are necessary). They will live at /platforms-new. Your team will have to create each new page (there are 4) with the proper page blocks while copy-pasting the content from the old pages onto the new ones.
3. **Once the new platform pages are created, your team should update the main navigation to use those URLs instead of the old ones.**
4. Then, we'll deploy an update that will turn off the ability for the platform taxonomy to have pages, remove the custom fields, and change the root slug on the new post type from /platforms-new to /platforms.

**Comment by ten1seven** [comment]
Re: #3, I think we can take the responsibility of changing the nav links as part of the deploy. That way we can do it right at the time of deploy to essentially prevent `platforms-new` from ever being indexed by Google, which could happen if those new links are up for even a day.

**Comment by haleyjohnso** [comment]
Great, I'll just remove step three then and roll the details into our deploy step. Thanks!

**Comment by haleyjohnso** [comment]
Whitney's team will start content entry on Tuesday at 1:30pm ET /11:30am MT. Can we deploy this to production before then? Thanks!
